sources/packages/backend/workflow/test/2025-2026/parttime-assessment/costs/parttime-assessment-costs-program-related-costs.e2e-spec.ts

```diff
@@ -8,13 +8,12 @@ import {
 describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-program-related-costs.`, () => {
   it(
     "Should get program related costs as offering program related costs when the DMN limit for books and supplies " +
-      "multiplied by offering weeks has the greater value.",
+      "minus the program year total part-time books and supplies cost has the greater value.",
```

Comment:
The "has the greater value." is lost at the end and makes it harder to read.
Should we create a second E2E based on this to check the scenario when there is a value for the PY total?